consensus/poet/core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -109,27 +109,43 @@ def compare_forks(self, cur_fork_head, new_fork_head):
                 block=new_fork_head,
                 poet_enclave_module=poet_enclave_module)
 
+
         # If we ever get a new fork head that is not a PoET block, then bail
         # out.  This should never happen, but defensively protect against it.
         if new_fork_wait_certificate is None:
+            print("new_fork_wait_certificate is ", new_fork_wait_certificate)
+            print("code reached")
             raise \
                 TypeError(
                     'New fork head {} is not a PoET block'.format(
                         new_fork_head.identifier[:8]))
+        # return ('new_fork_wait_certificate is None')
 
         # Criterion #1: If the current fork head is not PoET, then check to see
         # if the new fork head is building on top of it.  That would be okay.
         # However if not, then we don't have a good deterministic way of
         # choosing a winner.  Again, the latter should never happen, but
         # defensively protect against it.
+        # print("testing in compare_forks")
+        # print("new_fork_wait_certificate is: ", new_fork_wait_certificate)
+        # print("current_fork_wait_certificate is: ", current_fork_wait_certificate)
+
         if current_fork_wait_certificate is None:
+            # print("current_fork_wait_certificate is: ", current_fork_wait_certificate)
+            # print("new_fork_head.previous_block_id is ", new_fork_head.previous_block_id)
+            # print("cur_fork_head.identifier ", cur_fork_head.identifier)
+
             if new_fork_head.previous_block_id == cur_fork_head.identifier:
+                print("new_fork_head.previous_block_id == cur_fork_head.identifier")
                 LOGGER.info(
                     'Choose new fork %s: New fork head switches consensus to '
                     'PoET',
                     new_fork_head.identifier[:8])
                 chosen_fork_head = new_fork_head
+                print("chosen_fork_head is ", chosen_fork_head)
```

Comment:
These should be `LOGGER.debug` statements, or removed if they don't warrant that.